codec_specs.md

```diff
@@ -58,6 +58,18 @@ A timestamp, expressed in [@!RFC3339] that notes when support for the `Codec Map
 
 A `Codec Mapping` MAY only be defined with a `Superseded By` value, if it has an expressed `Deprecation Date`. If used, the `Superseded By` value MUST store the `Codec ID` of another `Codec Mapping` that has superseded the `Codec Mapping`.
 
+## Recommendations for the Creation of New Codec Mappings
+
+Creators of new `Codec Mappings` to be used in the context of Matroska:
+
+- SHOULD assume that all `Codec Mappings` they create might become standardized, public, commonly deployed, or usable across multiple implementations.
+
+- SHOULD employ meaningful values for `Codec ID` and `Codec Name` that they have reason to believe are currently unused.
+
+- SHOULD NOT prefix their `Codec ID` with "X-" or similar constructs.
```

Comment:
Should this not read `X_`? (underline, not dash)

Reply:
Not sure. For instance if an organization using Matroska starts using their own `V_EXPERIMENTAL/VP123`, then a year later proposes it for official support, it doesn't make sense to leave that experimental name as is, but we'd likely prefer `V_VP123`. If the original users simply used `V_VP123` from the beginning and the experimental nature was just evident since it wasn't initially in the Codec Mapping, then there wouldn't be an issue with changing Codec IDs.